buildpipeline/DotNet-CoreFx-Trusted-Windows-NoTest.json

```diff
@@ -1,5 +1,28 @@
 {
   "build": [
+    {
+      "environment": {},
+      "enabled": true,
+      "continueOnError": true,
+      "displayName": "run begin.ps1",
+      "timeoutInMinutes": 0,
+      "alwaysRun": true,
+      "task": {
+        "id": "e213ff0f-5d5c-4791-802d-52ea3e7be1f1",
+        "versionSpec": "2.*",
+        "definitionType": "task"
+      },
+      "inputs": {
+        "targetType": "inline",
+        "filePath": "",
+        "arguments": "",
+        "script": "if (Test-Path \"$(AgentToolsPath)\\begin.ps1\") {\n   \"begin.ps1 script found. Executing...\"\n   cd $(AgentToolsPath)\n   & $(AgentToolsPath)\\begin.ps1\n} else {\n   \"begin.ps1 script does not exist. Moving on...\"\n}",
```

Comment:
Inconsistency with begin/end.sh where here you echo found script and executing switching directories. I think it is a good idea to echo executing script with the location to the script and probably avoid switching directories.

Reply:
Thanks. Done.